docs/how-to/integrate-with-iam.md

```diff
@@ -0,0 +1,37 @@
+# How to integrate with IAM
+
+This charm supports integration with the [IAM bundle](https://charmhub.io/iam) via [Oathkeeper](https://charmhub.io/oathkeeper), adding an authentication layer that will front the Jenkins applications. When enabled, the [reverse-proxy-auth-plugin](https://plugins.jenkins.io/reverse-proxy-auth-plugin/) will be installed and configured, blocking unaothorised access to Jenkins.
+
+The steps to enable this mechanism are described below.
+
+## Deploy the IAM bundle
```

Comment:
I'm wondering if numerical headings could help since these are steps :)

Reply:
There are some other analogous cases where this is the format that we follow, so I think consistency should be taken into account. We can discuss this for general guidance :)